main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -132,17 +136,18 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
     input.suggest(suggestOptions).on("fb-select", function(e, data) {
         var types = data.notable ? data.notable : [];
 
-        Refine.postCoreProcess(
-        "recon-match-specific-topic-to-cells",
+        Refine.postOperation(
         {
+            op: "core/recon-match-specific-topic-to-cells",
             columnName: column.name,
-            topicID: data.id,
-            topicName: data.name,
-            types: types.join(","),
+            match: {
```

Comment:
Are the parameters changing here? It's hard to tell from the diff without digging into the old function.

Reply:
See the explanation below: this change comes from the difference between the encoding of the operation parameters as POST parameters of the operation command, and as a JSON object in the internal representation of the operation.